src/optimized/WormholeL2GovernorMetadataOptimized.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import {L2GovernorMetadata} from "src/L2GovernorMetadata.sol";
+import {WormholeL2GovernorMetadata} from "src/WormholeL2GovernorMetadata.sol";
+
+contract WormholeL2GovernorMetadataOptimized is WormholeL2GovernorMetadata {
+  /// @notice The internal proposal ID which is used by calldata optimized cast methods.
+  uint16 internal _nextInternalProposalId = 1;
```

Comment:
nit: our typical convention is to use a leading underscore for local vars, so this confuses my brain. Can we remove the `_` prefix? (feel fee to self resolve this issue and merge).

Reply:
Ah okay will do

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/69/commits/d9a8b9c6416bdc4ff398625c197cb70b093fb539